src/Contrib/Data/IdGenerator/Snowflake/Tests/Masa.Contrib.Data.IdGenerator.Snowflake.Tests/IdGeneratorTest.cs

```diff
@@ -17,10 +18,7 @@ public class IdGeneratorTest
     public void TestEnableMachineClock()
     {
         var services = new ServiceCollection();
-        services.AddSnowflake(opt =>
-        {
-            opt.EnableMachineClock = true;
-        });
+        services.AddSnowflake(opt => { opt.EnableMachineClock = true; });
```

Comment:
Line 37:Add at least one assertion to this test case.

Reply:
Processed